index.js

```diff
@@ -39,7 +36,15 @@ arrayPromise.then(responseArray => {
 
     let image = document.createElement('img');
     image.src = item.url;
-    image.alt = `${item.url}-photo`;
+    image.alt = `${item.id}-photo`;
+    image.setAttribute(`id`, `${item.id}`);
     frame.appendChild(image);
   })
-});
\ No newline at end of file
+
+
+});
+
+document.addEventListener('click', function (e) {
+  sessionStorage.setItem(`id`, `${e.target.id}`);
+});
+sessionStorage.setItem(`array`, `${arrayPromise}`);
```

Comment:
you want to make every item of yours clickable, so why not add onClickListener when u dynamically create the html above ? This is not a good idea or practice